### DISCUSSION TITLE
Cannot Load Up My UI Please Help

### DISCUSSION THREAD

**Discussion by dmnitro** [telekom-security/tpotce#1064] [General]
My T-Pot is up and running for the past 12 hours and for some odd reason I cannot get my UI to come up through the web URL with my IP and port. Using https://ip:64297 tried 64295 & 64294 as well
![image](https://user-images.githubusercontent.com/77514529/162674253-4de35a5a-25fe-4ff2-9e73-ab495572403e.png)
![image](https://user-images.githubusercontent.com/77514529/162675231-84ce775a-7700-415c-898d-e89633ced82a.png)

**Comment by tadashi-oya** [comment]
Can you connect to your T-Pot instance via ssh?
If you can connect via ssh, I suggest you try to include "Basic support information" in this thread.
https://github.com/telekom-security/tpotce/blob/master/.github/ISSUE_TEMPLATE/general-issue-for-t-pot.md

**Reply by dmnitro** [reply]
Looks like using ssh works but its just logs my into the VM instance i am already running 
![image](https://user-images.githubusercontent.com/77514529/162800402-8530ccf9-8d84-4467-9b12-2915efbc599d.png)

**Reply by tadashi-oya** [reply]
I suggest you try to write "Basic support information" in this thread.
https://github.com/telekom-security/tpotce/blob/master/.github/ISSUE_TEMPLATE/general-issue-for-t-pot.md

And please do not use screenshot, but
\`\`\`
use Code style
\`\`\`
```
use Code style
```

**Reply by dmnitro** [reply]
What version of the OS are you currently using?
lsb_release - No LSB modules are available 
Distributor ID: Debian
Description: Debian GNU/Linux 10 (Buster)
Release: 10
Codename: Buster

Uname -a 
Linux rapidpint 4.19.0.20-amd64 #1 SMP Debian 4.19.236-1 (2022-03–17) x86_64 GNU/Linux

What T-Pot version are you currently using?
20.06.2
 
What edition (Standard, Nextgen, etc.) of T-Pot are you running?
Standard used 
 
What architecture are you running on (i.e. hardware, cloud, VM, etc.)?
Macbook Windows Bootcamp 16gb RAM 256SSD - VM Vultr
 
How long has your installation been running?
2 days -  failed downloading a few times before since I had to download nodeJS according to logs and it kept running through the installation process and then returning me to the configuration menu. Started a new VM and it fixed the problem.
 
Did you install upgrades, packages or use the update script?
No
 
Please provide a screenshot of glances and htop.
![image](https://user-images.githubusercontent.com/77514529/163021168-94f8abf8-65b9-4f6c-bbf7-9bd7bf60422b.png)
Not sure how to view htop

What is the current container status (dps.sh)?
![image](https://user-images.githubusercontent.com/77514529/163021406-fb34f7c1-85c0-423d-9ce1-2b5a073662aa.png)


What is the status of the T-Pot service (systemctl status tpot)?
![image](https://user-images.githubusercontent.com/77514529/163021452-e5cc213a-3ad7-49d2-852d-d60340157150.png)

What ports are being occupied? Stop T-Pot systemctl stop tpot and run netstat -tulpen
![image](https://user-images.githubusercontent.com/77514529/163021611-3ed99382-3c5a-4b82-83b9-3050d43284b0.png)

Not very sure how to export the txt file created in the VM

**Reply by t3chn0m4g3** [reply]
From the info I can only assume that the NIC is either not bridged or exposed to the host / network.
Also recommending to install 22.04.

**Reply by dmnitro** [reply]
> From the info I can only assume that the NIC is either not bridged or exposed to the host/network. Also recommending to install 22.04.

How would I go about bridging the network and also upgrading to 22.04 without losing any of my progress if possible?
If there is a forum or link or you could let me know that would be appreciated thank you.

**Reply by dmnitro** [reply]
I managed to upgrade the version and get it running however now it seems the tpot only runs when im on the VM itself and not in the background also Blackhole is disabled any suggestions?
The only thing i can think of is because before i was working with just a CDL shell whereas now im working in the debian 11 OS terminal
I also tried connecting to the VM with Putty and kept getting 'remote side unexpectedly closed network connection'

**Reply by t3chn0m4g3** [reply]
I have never used Vultr VM before, so I do not how to advise here.
Blackhole is supposed to be disabled by default and should only be enabled if you desire a more stealth setup resulting in less events / attacks.
